easybuild/easyconfigs/p/PETSc/PETSc-3.7.5-intel-2016b-downloaded_deps.eb

```diff
@@ -0,0 +1,43 @@
+name = 'PETSc'
+version = '3.7.5'
+versionsuffix = '-downloaded_deps'
+
+homepage = 'http://www.mcs.anl.gov/petsc'
+description = """PETSc, pronounced PET-see (the S is silent), is a suite of data structures and routines for the
+ scalable (parallel) solution of scientific applications modeled by partial differential equations."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+toolchainopts = {'usempi': True, 'pic': True}
+
+source_urls = ['http://ftp.mcs.anl.gov/pub/petsc/release-snapshots']
+sources = [SOURCELOWER_TAR_GZ]
+
+builddependencies = [('CMake', '3.5.2')]
+shared_libs = 1
+pic = 1
```

Comment:
`pic` does not exist as an easyconfig parameter, neither does `downloadinstall`

fixed in https://github.com/ComputeCanada/easybuild-easyconfigs/pull/2, along with some additional cleanup